articles/data-factory/copy-activity-log.md

```diff
@@ -99,6 +99,8 @@ After the copy activity runs completely, you can see the path of log files from
         }
 
 ```
+> [!NOTE]
+> It isn't possible to provide a user-defined name for the log files which are generated by enabling the `enableCopyActivityLog` property.
```

Comment:
The note is useful. What you can do is to update it as "When the 'enableCopyActivityLog' property is set to Enabled, the log file names are system generated."

Reply:
Thank you @dcstwh for the comment.
I have made the changes as per review comment